products/moodle.md

```diff
@@ -19,13 +19,15 @@ releases:
     releaseDate: 2023-10-07
     support: 2024-10-14
     eol: 2025-04-14
+    supportedPhpVersions: 8.0 - 8.2
     latest: "4.3.0"
     latestReleaseDate: 2023-10-07
 
 -   releaseCycle: "4.2"
     releaseDate: 2023-04-22
     support: 2024-04-22
     eol: 2024-10-07
+    supportedPhpVersions: (4.2.0-4.2.2) 8.0 - 8.1, (4.2.3~) 8.0 - 8.2
```

Comment:
8.0 - 8.2 would be sufficient / much clearer IMO, the reasoning behind that being:

- one should always use the latest patch version, particularly if he plan to upgrade PHP version,
- endoflife.date is mostly about release cycles and not about specific versions.

WDYT @topstone ?

Reply:
I prefer simpler description.

@marcwrobel , How about [this variant](https://github.com/topstone/endoflife.date_moodle/blob/PHP_table_variant/products/moodle.md)?

Reply:
Looks good !

Reply:
Thanks! I'll close this PR and make another PR.